Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Do you mind stripping out the Mozilla License comment block and assuming the license on the root directory the project (which is Mozilla License 2.0)? It just to simplify the licensing, so if the license ever need to be updated, we only need to worry about updating one location.